hangman_dinner.rb

```diff
@@ -0,0 +1,201 @@
+      @current_answer_array = @current_answer.split(//)
+      @fillable_answer = []
+      @alphabet = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K","L", "M", "N", "O", "P", "Q", "R", "S",
+      "T", "U", "V", "W", "X", "Y", "Z"]
+      @already_guessed =[]
+      @current_answer_array.length.times do |x|
+        @fillable_answer[x]= " _ "
+      end
+
+
+      draw_art
+      print_word
+      play
+
+    end
+
+    def draw_art
+
+
+      art = <<-END
+|     _________
+|     |/      |
+|     |
+|     |
+|     |
+|     |
+|     |
+|_____|___________
+
+  END
+
+    if @guesses == 3
+      head = "(_)".colorize(:red)
+      art = <<-END
+
+      |     _________
+      |     |/      |
+      |     |      #{head} "I would wave for help but I don't have arms."
+      |     |
+      |     |
+      |     |
+      |     |
+      |_____|___________
+
+        END
+
+      elsif @guesses == 2
+        head = "(_)".colorize(:red)
+        arms = "\\|/".colorize(:white)
+        art = <<-END
+
+        |     _________
+        |     |/      |
+        |     |      #{head} "Is it getting hot in here? Anyone?"
+        |     |      #{arms}
+        |     |
+        |     |
+        |     |
+        |_____|___________
+
+          END
+
+        elsif @guesses == 1
+          head = "(_)".colorize(:red)
+          arms = "\\|/".colorize(:white)
+          torso = "|".colorize(:blue)
+          art = <<-END
+
+          |     _________
+          |     |/      |
+          |     |      #{head} "Lietuenent Dan: I can't feel my legs!"
+          |     |      #{arms}
+          |     |       #{torso}
+          |     |
+          |     |
+          |_____|___________
+
+            END
+
+          elsif @guesses == 0
+            head = "(_)".colorize(:red)
+            arms = "\\|/".colorize(:white)
+            torso = "|".colorize(:blue)
+            legs =  "/ \\".colorize(:magenta)
+          art = <<-END
+          |     _________
+          |     |/      |
+          |     |      #{head} "Dead men tell no tales!"
+          |     |      #{arms}
+          |     |       #{torso}
+          |     |      #{legs}
+          |_____|___________
+
+            END
+
+          end
+
+          puts art
+    puts @current_answer
+    #drawing hangman
+    end
+
+    def print_word
+      print "WORD: "
+
+
+      puts @fillable_answer.join
+      puts ""
+      puts "Letters Guessed: #{@already_guessed.join(" ")}"
+      puts "Incorrect Guesses Left: #{@guesses}"
+      puts ""
+    end
+
+    def play
+      while @guesses > 0
+        puts "Type a letter or word"
+        letter = gets.chomp.downcase
+
+        wrong = true
+        if letter.length == 1
+        until @alphabet.include?(letter.upcase)
+        puts "You must put an actual letter or word please"
+        puts "Type a letter or word"
+        letter = gets.chomp.downcase
+      end
+
+        @current_answer_array.length.times do |x|
```

Comment:
Instead of looping through every letter of the answer, we could refactor this to be a little faster by using `include?` to check for `current_answer_array` contains input letter.